config/servicemappings/alloydb.yaml

```diff
@@ -115,6 +114,18 @@ spec:
             version: v1beta1
             kind: ComputeNetwork
           targetField: self_link
+        - tfField: network_config.network
```

Comment:
Could you help me understand that why it is external only?

Reply:
It was based on the testing from AlloyDB team. It may be a bug in their API which needs to be fixed. Sent you a related link separately.